lib/carbon/instrumentation.py

```diff
@@ -23,6 +25,7 @@
 # more consistent, and make room for frontend metrics.
 #metric_prefix = "Graphite.backend.%(program)s.%(instance)s." % settings
 
+DATAPOINTS_RECEIVED = Counter('datapoints_received_total', 'count of datapoint received by carbon agent')
```

Comment:
should we consider renaming these with `carbon_` prefix

Reply:
That is a good point, I'll add it for the  datapoints received as this metric is specific to carbon